library/itertools.po

```diff
@@ -825,27 +823,27 @@ msgid ""
 "sorted order."
 msgstr ""
 "Les boucles imbriquées tournent comme un compteur kilométrique avec "
-"l'élément le plus à droite avançant à chaque itération. ce motif créé un tri "
-"lexicographique afin que si les itérables de l'entrée sont triés, les "
-"*tuples* de produit sont émis dans l'ordre."
+"l'élément le plus à droite avançant à chaque itération. Ce motif crée un tri "
+"lexicographique afin que si les itérables de l'entrée sont triés, les n-"
```

Comment:
```suggestion
"lexicographique afin que, si les éléments des itérables en entrée sont classés, les n-"
```

Reply:
Je voit toujours `afin que si les itérables de l'entrée`